subsys/usb/class/netusb/function_ecm.c

```diff
@@ -27,16 +27,12 @@
 #define USB_CDC_ECM_REQ_TYPE		0x21
 #define USB_CDC_SET_ETH_PKT_FILTER	0x43
 
-struct ecm {
-	struct net_if *iface;
-
-	/* In a case of low memory skip data to the end of the packet */
-	bool skip;
-} ecm;
-
 /* Pointer to pkt assembling at the moment */
 struct net_pkt *in_pkt;
```

Comment:
Shouldn't this be static?

Reply:
Yes, this is my fault, should be static.